src/RuleSet/Sets/PhpCsFixerRiskySet.php

```diff
@@ -25,18 +25,6 @@ public function getRules()
             '@Symfony:risky' => true,
             'comment_to_phpdoc' => true,
             'final_internal_class' => true,
-            // @TODO: consider switching to `true`, like in @Symfony
```

Comment:
the decision for this action was not yet done.
merging this PR as-is will cause a lot of conflicts between 2.x <> 3.x and lead to conflicts on PRs too

Reply:
also, default config for this rule changed in v3: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/3.0/UPGRADE-v3.md#changed-default-values-of-options

Reply:
In Symfony:risky ruleset this was changed in 2.x. Thus I propose to change PhpCsFixer:risky ruleset in 2.x as well.

Reply:
I understood that intention from PR title,

unfortunately, your comment does not address my concerns

Reply:
Can I help addressing your concerns? You want this change in 2.x or 3.x only?

Reply:
Due to amount of conflicts it gonna create for us, syncing between v2<>v3 and all open PRs, I don't see big value of having this change merged right now.
What would be the benefit overcoming the downsides?

Reply:
For these reasons, I opened this PR into v2. I belive, FQCN are important as they can be optimized at compile time.

Reply:
it does not solve the concern for conflicts for all open PRs.

Let's reject the PR for now and re-evaluate this change after we decrease amount of open PRs or stop v2 support